lib/pbench/server/__init__.py

```diff
@@ -52,16 +52,26 @@ def dst(self, dt) -> timedelta:
         return timedelta(0)
 
 
+UTC = SimpleUTC()
+
+
+def utcnow(ts: float = None) -> datetime:
+    """Convenience method for obtaining a datetime object with a UTC time zone.
+
+    Returns:
+        A datetime object with a UTC time zone.
+    """
+    ts = _time() if ts is None else ts
+    return datetime.utcfromtimestamp(ts).replace(tzinfo=UTC)
```

Comment:
This is good enough, but we might have wished for a description of the argument in the docstring.

Also, although it's lovely that it fits on one line, the ternary doesn't serve us well in that we don't need to assign the value to itself.  Moreover, since we conventionally prefer not to overwrite parameters, something more like this would be better:
```
    tv = _time() if ts is None else ts
    return datetime.utcfromtimestamp(tv).replace(tzinfo=UTC)
```

Reply:
Addressed both, see what you think.